### PR TITLE
Fix index.ejs not copied to dist folder

### PR DESCRIPTION
**Pull request by hieptuanle**
Fix issue #65 

This PR just copy `index.ejs` file from `src/ui` to `dist/ui`. I add a new library `cpx` because I don't want to write a bash script and mess with Windows users.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,6 +67,7 @@
     "babel-preset-react-app": "^7.0.2",
     "bull": "^3.12.1",
     "bullmq": "^1.6.6",
+    "cpx": "^1.5.0",
```
Comment (erikengervall) [inline package.json]: Couldn't find any documentation that this indeed supports windows, have you tried it yourself?

Also the docs state

> Copy file globs, watching for changes.
> 
> This module provides a CLI tool like cp, but with watching.

We're not really interested in watching for changes - could this have unforeseen side-effects? Would perhaps be better with a more plain file-copy lib.

**Comment by vcapretz** [comment]
hey @hieptuanle, thank you very much for taking the time to contribute to the project, really appreciate it 😄 

I just merged #82 and the same issue was solve there, so I'll close your PR

if you want to contribute more I would be very glad to review more PRs 👍

File: package.json (hunk 1 above)
Comment (hieptuanle) [reply package.json]: Actually i did not test it on Windows. I will test it next time 👍

**Comment by hieptuanle** [comment]
@vcapretz No problems! Thanks for your useful project.

**Pull request closed without merging**